src/main/java/frc/robot/lib/interfaces/Swerve.java

```diff
@@ -122,6 +124,14 @@ public static void periodic() {
                 rotation
             ));
         }
+
+        for (int i = 0; i < 2; i++) {
+            loadingStationPoses.add(new Pose2d(
+                starting_loading_station.getX(),
+                Constants.TargetPoses.BLUE_SUBSTATION_Y + i * Constants.TargetPoses.SUBSTATION_SPACING,
+                rotation
```

Comment:
I do not think this rotation is correct. Remember, the scoring position rotation and the substation rotation should be 180 degrees opposite eachother.